tests/code_feature_test.py

```diff
@@ -6,15 +6,20 @@
 
 def test_split_file_into_intervals(temp_testbed, mock_session_context):
     with open("file_1.py", "w") as f:
-        f.write(dedent("""\
+        f.write(
+            dedent(
+                """\
             def func_1(x, y):
                 return x + y
             
             def func_2():
                 return 3
-            """))
+            """
+            )
+        )
```

Comment:
Black did this for me automatically as this file wasn't formatted earlier.

Reply:
When I run `black --preview .`, it just changes this back, and no other reformats [🤔](https://emojipedia.org/thinking-face)

Reply:
Oof might be something on my end then

Reply:
Ah looks like I wasn't running black in "preview" mode